scout/adapter/mongo/case.py

```diff
@@ -689,7 +689,11 @@ def update_case(self, case_obj, keep_date=False):
         The following will be updated:
             - collaborators: If new collaborators these will be added to the old ones
             - analysis_date: Is updated to the new date
-            - analyses: The new analysis date will be added to old runs
+            - delivery_report: path to the static delivery report file
```

Comment:
I've rearranged this comment to reflect the order of the items that are updated later in the code. Some items where missing from here.

Reply:
Good! Ideally reorder with models and/or alphabet, but it's all good otherwise.